test/src/e2e.dynval/dv.n'.test.ts

```diff
@@ -17,37 +17,26 @@
 import * as chai from "chai";
 import { expect } from "chai";
 import * as chaiAsPromised from "chai-as-promised";
-import { blake256, H256 } from "codechain-primitives/lib";
-import { SDK } from "codechain-sdk";
 import * as stake from "codechain-stakeholder-sdk";
 import "mocha";
 
 import { validators as originalDynValidators } from "../../tendermint.dynval/constants";
 import {
-    aliceSecret,
-    bobSecret,
     faucetAddress,
     faucetSecret,
-    stakeActionHandlerId,
-    validator0Address,
-    validator1Address,
-    validator2Address,
-    validator3Address
 } from "../helper/constants";
-import { PromiseExpect, wait } from "../helper/promise";
-import CodeChain from "../helper/spawn";
+import { PromiseExpect } from "../helper/promise";
 import { withNodes } from "./setup";
 
 chai.use(chaiAsPromised);
 
-const RLP = require("rlp");
-
-// alice : Elected as a validator, but does not send precommits and does not propose.
-//   Alice should be jailed.
-// betty : Not elected as validator because of small delegation. She acquire more delegation in the first term.
-//   betty should be a validator in the second term.
-const [alice, betty, notUsed, ...otherDynValidators] = originalDynValidators;
-const allDynValidators = [alice, betty, ...otherDynValidators];
+const [
+    alice,
+    betty,
+    ,
+    ,
+    ...otherDynValidators
+] = originalDynValidators;
 
 describe("Dynamic Validator N -> N'", function() {
     this.timeout(0);
```

Comment:
Would you set a proper timeout?
See: https://github.com/CodeChain-io/codechain/pull/1706